### ISSUE TITLE
Cannot find module 'bingmaps-react' or its corresponding type declarations.ts(2307)

### ISSUE THREAD

**Issue by sharief-acc** [milespratt/bingmaps-react#36]
Install the latest version 1.1.1 and getting the error --> Cannot find module 'bingmaps-react' or its corresponding type declarations.ts(2307)
import BingMapsReact from "bingmaps-react";

**Comment by aviboy2006** [comment]
package shows in node_modules ?

**Comment by jimmywim** [comment]
I think you're missing an index.js on your published npm package.

Gotta import with:

`import BingMapsReact from "bingmaps-react/BingMapsReact";`

**Comment by milespratt** [comment]
@sharief-acc @jimmywim are you both using typescript? This package wasn't built with typescript in mind but it's on the roadmap and I'm happy to look into the issue.

I just created a fresh react app with version 1.1.1 and am not having issues installing or using the package as follows:

`npm install bing maps-react`

```
import "./App.css";
import BingMapsReact from "bingmaps-react";

function App() {
  return (
    <div className="App">
      <BingMapsReact />
    </div>
  );
}

export default App;
```

node_modules shows the files as expected:

<img width="226" alt="Screen Shot 2021-12-02 at 10 45 10 AM" src="https://user-images.githubusercontent.com/45855051/144454771-6cff1a29-edd4-4119-9311-fb4d2f35ce0a.png">

**Comment by milespratt** [comment]
I just published a new version, 1.1.4, that may resolve your issues. Please let me know.

**Comment by milespratt** [comment]
@sharief-acc @jimmywim I added some simple TS support via some minimal declarations. I expect this to work in TS apps without any hassle now.
